toolchain.mk

```diff
@@ -1,6 +1,7 @@
 ################################################################################
 # Toolchains
 ################################################################################
+SHELL				= /bin/bash
```

Comment:
maybe `?=` to inherit from `SHELL` is already defined?

Reply:
> maybe `?=` to inherit from `SHELL` is already defined?

Not sure, lots of our stuff depends on bash. I.e., the shell isn't really an option for OP-TEE makefiles I guess.

Reply:
> > maybe `?=` to inherit from `SHELL` is already defined?
> 
> Not sure, lots of our stuff depends on bash. I.e., the shell isn't really an option for OP-TEE makefiles I guess.

Yes not a good idea, see https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html:
*"Unlike most variables, the variable SHELL is never set from the environment. This is because the SHELL environment variable is used to specify your personal choice of shell program for interactive use. It would be very bad for personal choices like this to affect the functioning of makefiles."*

Reply:
ok, understood. thanks.

Reply:
So , do I need modify?

Reply:
> So , do I need modify?

No, please keep it as it is, thanks!

Reply:
Sorry for the noise @liuyunli.